app/helpers/bulk_action_helper.rb

```diff
@@ -9,6 +9,12 @@ def render_bulk_action_type(bulk_action)
     render partial: 'default'
   end
 
+  # link back to the search results page, and remove the pids_only param from the search context
+  def back_to_search_results
+    current_search_session.try(:query_params)&.delete('pids_only')
```

Comment:
why do you need a `try` here?

Reply:
Would it be easier to not store `pids_only` in the search history?  That would involve overwriting ` find_or_initialize_search_session_from_params` on the controller to call super with params.except('pids_only')`   I can't think of any time where we'd want that.

Reply:
I think I found a better way using this method I just found in blacklight while poking around: https://github.com/projectblacklight/blacklight/blob/master/app/controllers/concerns/blacklight/search_context.rb#L104-L107

Reply:
Testing in argo-qa, seems to work fine.